src/utils/image-processor.js

```diff
@@ -1,37 +1,42 @@
 import $ from 'jquery'
 
 import { Blob, FileReader, URL } from '../utils/abilities'
-import { imageLoader } from '../utils/image-loader'
+import { imageLoader } from '../utils/image-loader.ts'
 import { defer, fitSize, canvasToBlob, taskRunner } from '../utils'
 import { isWindowDefined } from './is-window-defined'
 
 // utils image
-var DataView = isWindowDefined() && window.DataView
-var runner = taskRunner(1)
+const DataView = isWindowDefined() && window.DataView
+const runner = taskRunner(1)
 
 const shrinkFile = function(file, settings) {
+  let res = () => {}
```

Comment:
`error  'res' is assigned a value but never used  @typescript-eslint/no-unused-vars`

Please fix